gtk4/sys/Cargo.toml

```diff
@@ -7,7 +7,6 @@ edition = "2018"
 homepage = "http://gtk-rs.org/"
 keywords = ["gtk", "gtk4", "ffi", "gtk-rs", "gnome"]
 license = "MIT"
-links = "gtk-4"
```

Comment:
Why is this a problem?

Reply:
Without this change i get 
```
the package `gtk4-sys` links to the native library `gtk-4`, but it conflicts with a previous package which links to `gtk-4` as well:
package `gtk4-sys v0.1.0 (/__w/gtk4-rs/gtk4-rs/gtk4/sys)`

failed to select a version for `gtk4-sys` which could resolve this conflict
```
dropping the usage of links in the other -sys crates in gtk3-rs/gtk-rs-core fixes their release CI as well

Reply:
Which other crate lists `gtk-4` as `links`? This error should only happen if there are two or more.

Reply:
there are none of them actually, only the -sys crates do

Reply:
That must mean that the crate is pulled in multiple times. Please debug this instead of removing the `links=` sections :) Something's very wrong.